plugins/systemstat.koplugin/main.lua

```diff
@@ -120,26 +129,21 @@ function SystemStat:appendSystemInfo()
         self:put({_("System information"), ""})
         -- @translators Ticks is a highly technical term. See https://superuser.com/a/101202 The correct translation is likely to simply be "ticks".
         self:put({_("  Total ticks (million)"),
-                 string.format("%.2f", stat.cpu.total / 1000000)})
+            string.format("%.2f", stat.cpu.total / 1000000)})
         -- @translators Ticks is a highly technical term. See https://superuser.com/a/101202 The correct translation is likely to simply be "ticks".
         self:put({_("  Idle ticks (million)"),
-                 string.format("%.2f", stat.cpu.idle / 1000000)})
-        self:put({_("  Processor usage %"),
-                 string.format("%.2f", (1 - stat.cpu.idle / stat.cpu.total) * 100)})
+            string.format("%.2f", stat.cpu.idle / 1000000)})
+        self:put({_("  Processor usage"),
+            string.format("%.2f %%", (1 - stat.cpu.idle / stat.cpu.total) * 100)})
     end
     if stat.memory ~= nil then
-        if stat.memory.total ~= nil then
-            self:put({_("  Total memory (MB)"),
-                     string.format("%.2f", stat.memory.total / 1024)})
-        end
-        if stat.memory.free ~= nil then
-            self:put({_("  Free memory (MB)"),
-                     string.format("%.2f", stat.memory.free / 1024)})
-        end
-        if stat.memory.available ~= nil then
-            self:put({_("  Available memory (MB)"),
-                     string.format("%.2f", stat.memory.available / 1024)})
-        end
+        self:put({_("  Total") .. SEP .. _("free") .. SEP .. _("available"),
```

Comment:
I think this needs the word "memory" or "RAM", because otherwise, it's not obvious what `Total . free . available`: is.
(After a few seconds, one must have guessed what it is, but it would be better if it just reads what it is.)